recipes/s2n/all/test_package/test_package.c

```diff
@@ -0,0 +1,9 @@
+#include <aws/compression/compression.h>
```

Comment:
Needed for `EXIT_SUCCESS`.
```suggestion
#include <aws/compression/compression.h>

#include <stdlib.h>
```